src/Infrastructure/Persistance/CommandHandlers/Users/LoginCommandHandler.cs

```diff
@@ -37,7 +37,9 @@ public async Task<LoginResponse> Handle(LoginCommand request, CancellationToken
                 throw new CleanArchAppException("username or password is incorrect");
 
             var jwt = await _jwtService.GenerateAsync(user);
-
+            user.RefreshToken = jwt.refresh_token;
```

Comment:
Please don't forget about unit tests.
First: run unit tests and make sure that every test passes.
Second: write unit tests for new commands and services, etc.